packages/studio/inspector-kit/story/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../../../tsconfig.packages.json",
+  "compilerOptions": {
+    "rootDir": "..",
+
+    // We use react
+    "jsx": "react",
+    "resolveJsonModule": true
+  },
+  "include": [
+    "../story/**/*",
+    "../test/**/*.json",
+    "../src/**/*",
+    // this should probably not be there
+    "../demo/**/*"
```

Comment:
Then remove it, same thing for the test files and we'll re-add them if necessary

Reply:
It needs to be there because we import stuff from the demo, this is just a note that this should be removed later